pkg/omf/cli/omf.bundle.install.fish

```diff
@@ -7,24 +7,24 @@ function omf.bundle.install
     set bundle_contents (cat $bundle | sort -u)
 
     for record in $bundle_contents
-      set type (echo $record | cut -d' ' -f1)
-      set name_or_url (echo $record | cut -d' ' -f2-)
-      set name (basename $name_or_url | sed 's/\.git//;s/^pkg-//;s/^plugin-//;s/^theme-//')
+      test -n "$record"; or continue
+
+      set type (echo $record | cut -s -d' ' -f1 | sed 's/ //g')
+      contains $type theme package; or continue
 
-      switch $type
-      case "package"
-        if not contains $name $packages
-          omf.install --pkg $name_or_url
-        end
+      set name_or_url (echo $record | cut -s -d' ' -f2- | sed 's/ //g')
+      test -n "$name_or_url"; or continue
 
-      case "theme"
-        if not contains $name $themes
-          omf.install --theme $name_or_url
-        end
+      set name (basename $name_or_url | sed 's/\.git//;s/^pkg-//;s/^plugin-//;s/^theme-//')
+
+      if not contains $name $packages $themes
+        omf.install $name_or_url
```

Comment:
since `omf.install` has a useful status code, we could collect it here and change the `return 0` below so that `omf install` doesn't have to refresh unless it's necessary.